### PR TITLE
[shopsys] Sitemap updates

### PR DESCRIPTION
**Pull request by RostislavKreisinger**
| Q             | A
| ------------- | ---
|Description, reason for the PR| We want the data in the sitemap to be true and relevant, or rather not be there. All information about `<lastmod>`, `<changefreq>` and `<priority>` is removed from the sitemap (page and images). Added image sitemap
|New feature| Yes 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues|  
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: project-base/config/services/cron.yaml [hunk 1]
```diff
@@ -52,6 +52,10 @@ services:
         tags:
             - { name: shopsys.cron, hours: '4', minutes: '0', readableName: 'Generate Sitemap' }
 
+    Shopsys\FrameworkBundle\Model\ImageSitemap\ImageSitemapCronModule:
+        tags:
+            - { name: shopsys.cron, hours: '4', minutes: '10', readableName: 'Generate image sitemap' }
+
```
Comment (TomasLudvik) [inline project-base/config/services/cron.yaml:58]: You can squash second and third commit

**Review by TomasLudvik — CHANGES_REQUESTED** [review]

File: packages/framework/src/Model/Sitemap/SitemapListener.php [hunk 2]
```diff
@@ -17,6 +17,7 @@ class SitemapListener implements EventSubscriberInterface
     protected const PRIORITY_CATEGORIES = 0.8;
     protected const PRIORITY_PRODUCTS = 0.7;
     protected const PRIORITY_ARTICLES = 0.5;
```
[the remark this answers is not in the document]
Comment (RostislavKreisinger) [reply packages/framework/src/Model/Sitemap/SitemapListener.php]: ok

File: project-base/config/services/cron.yaml (hunk 1 above)
Comment (RostislavKreisinger) [reply project-base/config/services/cron.yaml:58]: ok, i do it before merge

File: packages/framework/src/Model/ImageSitemap/ImageSitemapCronModule.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+     * @var \Shopsys\FrameworkBundle\Model\ImageSitemap\ImageSitemapFacade
+     */
+    protected ImageSitemapFacade $imageSitemapFacade;
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\ImageSitemap\ImageSitemapFacade $imageSitemapFacade
+     */
+    public function __construct(ImageSitemapFacade $imageSitemapFacade)
+    {
+        $this->imageSitemapFacade = $imageSitemapFacade;
```
[the remark this answers is not in the document]
Comment (RostislavKreisinger) [reply packages/framework/src/Model/ImageSitemap/ImageSitemapCronModule.php]: ok

File: packages/framework/src/Model/Sitemap/SitemapListener.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (TomasLudvik) [reply packages/framework/src/Model/Sitemap/SitemapListener.php]: So what is the solution for this?

File: packages/framework/src/Model/Sitemap/SitemapListener.php [hunk 4]
```diff
@@ -13,39 +13,18 @@
 
 class SitemapListener implements EventSubscriberInterface
 {
-    protected const PRIORITY_HOMEPAGE = 1;
-    protected const PRIORITY_CATEGORIES = 0.8;
-    protected const PRIORITY_PRODUCTS = 0.7;
-    protected const PRIORITY_ARTICLES = 0.5;
```
Comment (TomasLudvik) [inline packages/framework/src/Model/Sitemap/SitemapListener.php:19]: You will have to mention this in the UPGRADE notes

**Review by TomasLudvik — APPROVED** [review]

**Pull request merged**